components/Alert.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from '../styles/components/alert.module.scss';
+import Button from './ui/Button';
+
+const Alert = ({ message, visible, onClose }) => (
+  <div
+    className={visible ? [styles.popUpWrapper, styles.show].join(' ')
+      : styles.popUpWrapper}
+  >
+    <div className={styles.popUp}>
+      <div className={styles.popUpHeader}>
+        <Button onClick={onClose}>+</Button>
+      </div>
+      <div className={styles.popUpMessage}>{message}</div>
+      <div className={styles.popUpFooter}>
+        <Button onClick={onClose}>OK</Button>
```

Comment:
Maybe the text on the button should be customizable as well? With a default value of `OK`.

Reply:
Maybe buttons should be castomizable?

Reply:
You mean the list of the buttons? We could do it later, as it's much more complex. But customizable text on the button is a quick task.